frontend/src/reducer/searchResult.js

```diff
@@ -16,16 +45,31 @@ const defaultResult = {
 }
 
 function uniqueResults (allResults) {
-  return uniqBy(allResults, 'content_id')
+  return uniqBy(allResults, 'contentId')
 }
 
 export default function searchResult (state = defaultResult, action) {
   switch (action.type) {
     case `${SET}/${SEARCH_RESULTS_LIST}`:
-      return { ...state, resultsList: uniqueResults(action.newSearchResultsList) }
+      return {
+        ...state,
+        resultsList: uniqueResults((action.newSearchResultsList).map(item => serialize(item, serializeSearchItemProps)))
```

Comment:
we usually don't put parenthesis around the array we are using .map() on.
There isn't any rules though